src/main/java/com/getindata/connectors/http/internal/utils/JavaNetHttpClientFactory.java

```diff
@@ -88,6 +91,21 @@ private static SSLContext getSslContext(Properties properties) {
         String clientPrivateKey = properties
             .getProperty(HttpConnectorConfigConstants.CLIENT_PRIVATE_KEY, "");
 
+        if (StringUtils.isNullOrWhitespaceOnly(keyStorePath)
+            && !selfSignedCert
+            && serverTrustedCerts.length == 0
+            && StringUtils.isNullOrWhitespaceOnly(clientCert)
+            && StringUtils.isNullOrWhitespaceOnly(clientPrivateKey)) {
+
+            try {
+                return SSLContext.getDefault();
+            } catch (NoSuchAlgorithmException e) {
+                throw new RuntimeException(e);
+            }
+        }
+
+        SecurityContext securityContext = createSecurityContext(properties);
```

Comment:
Talking with our team around this, in addition to the above. It would be good to have the ability to:
1- run with public certs and any supplied private server certs. i.e. merge the supplied certs with existing public ones.
2- run only with private server certs (which is the current functionality when they are supplied in the config) 

This would require a new config option maybe 'gid.connector.http.security.cert.server.includeDefaultCerts' with a default of true. 
 
WDYT?